docs/src/lib/representations.md

```diff
@@ -87,6 +87,15 @@ an_element(::Ballp{Float64})
 center(::Ballp{Float64})
 ```
 
+## Ellipsoid
+
+```@docs
```

Comment:
add `dim`?

Reply:
when i added dim for the ellipsoid, it printed the generic one for the `AbstractPointSymmetricPolytope`; that's why i removed it. see for example [hyperrectangle's dim](https://juliareach.github.io/LazySets.jl/latest/lib/representations.html#LazySets.dim-Tuple{LazySets.Hyperrectangle{Float64}})

Reply:
what shall we do?

Reply:
This is expected because we do not redefine it. I would say either we remove all functions that are inherited (in a separate PR) or add it here as well (to show that the function exists).

Reply:
ok. i don't like that there is a repeated docstring. maybe we should say it at the interface level if it's not said already.

Reply:
Good, I created #159 for this.